paper-drawer-panel.html

```diff
@@ -757,9 +766,43 @@
 
         _moveDrawer: function(translateX) {
           this.transform(this._transformForTranslateX(translateX), this.$.drawer);
-        }
+        },
+
+        _getAutoFocusedNode: function() {
+          return Polymer.dom(this).querySelector('[autofocus]') || this.$.firstTabStop;
```

Comment:
Is this intended to look for `[autofocus]` in distributed children? Doesn't ShadowDOM/Polymer.dom exclude distributed children from querySelector?